Source/Analytics/Events.Admin/HealthRisks/HealthRiskCreated.cs

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) The International Federation of Red Cross and Red Crescent Societies. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+using System;
+using Dolittle.Artifacts;
+using Dolittle.Events;
+
+namespace Events.Admin.HealthRisks
+{
+    [Artifact("51b2c376-ce2b-4d49-a86c-e654e50248c9", 1)]
+    public class HealthRiskCreated : IEvent
+    {
+        public Guid Id { get; }
```

Comment:
Should this be a HealthRiskId not a guid?

Reply:
No, "an event cannot contain a Concept" says Dolittle build tools 🙂